src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -384,19 +383,13 @@ public function send()
      *
      * Valid types are strings, numbers, null, and objects that implement a __toString() method.
      *
-     * @param mixed $content Content that can be cast to string
-     *
      * @return $this
      *
      * @throws \UnexpectedValueException
      */
-    public function setContent($content)
+    public function setContent(?string $content)
     {
-        if (null !== $content && !\is_string($content) && !is_numeric($content) && !\is_callable([$content, '__toString'])) {
-            throw new \UnexpectedValueException(sprintf('The Response content must be a string or object implementing __toString(), "%s" given.', \gettype($content)));
-        }
-
-        $this->content = (string) $content;
+        $this->content = $content;
```

Comment:
since `$content` is nullable, `$this->content` can also be null. this might break for people expecting string in `getContent`. so I guess we need to cast null to `''`

Reply:
there is actually a failing test due to this: https://travis-ci.org/symfony/symfony/jobs/569486612